### PR TITLE
Global new ignore review limit

### PR DESCRIPTION
**Pull request by RumovZ**
Update of #2398.

### CODE REVIEW

File: ftl/core/deck-config.ftl [hunk 1]
```diff
@@ -34,11 +34,17 @@ deck-config-limit-new-bound-by-reviews =
     shown.
 deck-config-limit-interday-bound-by-reviews =
     The review limit also affects interday learning cards. When applying the limit,
-    interday learning cards are fetched first, then reviews, and finally new cards.
```
Comment (RumovZ) [inline ftl/core/deck-config.ftl:37]: Do we need to add a new string for this?

File: rslib/src/ops.rs [hunk 2]
```diff
@@ -161,7 +161,11 @@ impl OpChanges {
         let c = &self.changes;
         (c.card && self.op != Op::SetFlag)
             || c.deck
-            || (c.config && matches!(self.op, Op::SetCurrentDeck | Op::UpdatePreferences))
+            || (c.config
+                && matches!(
+                    self.op,
+                    Op::SetCurrentDeck | Op::UpdatePreferences | Op::UpdateDeckConfig
+                ))
```
Comment (dae) [inline rslib/src/ops.rs:168]: Good catch

File: rslib/src/scheduler/queue/builder/mod.rs [hunk 3]
```diff
@@ -325,38 +332,27 @@ mod test {
...
     #[test]
     fn should_build_empty_queue_if_limit_is_reached() {
-        let mut col = open_test_collection();
-        col.set_config_bool(BoolKey::Sched2021, true, false)
-            .unwrap();
-        let note_id = col.add_new_note("Basic").id;
-        let cids = col.storage.card_ids_of_notes(&[note_id]).unwrap();
-        col.set_due_date(&cids, "0", None).unwrap();
+        let mut col = Collection::new_v3();
+        CardAdder::new().due_dates(["0"]).add(&mut col);
```
Comment (dae) [inline rslib/src/scheduler/queue/builder/mod.rs:336]: Nice!

**Review by dae — COMMENTED** [review]
Thanks Rumo, looks great.

File: rslib/src/import_export/package/apkg/import/decks.rs [hunk 4]
```diff
@@ -210,24 +210,23 @@ mod test {
...
         let mut ctx = DeckContext::new(&mut col, Usn(1));
         ctx.unique_suffix = "★".to_string();
 
         let imports = vec![
-            new_deck_with_machine_name("unknown parent\x1fchild", false),
-            new_deck_with_machine_name("filtered\x1fchild", false),
-            new_deck_with_machine_name("parent\x1fchild", false),
-            new_deck_with_machine_name("NEW PARENT\x1fchild", false),
-            new_deck_with_machine_name("new parent", false),
+            DeckAdder::new("unknown parent\x1fchild").deck(),
```
[the remark this answers is not in the document]
Comment (RumovZ) [reply rslib/src/import_export/package/apkg/import/decks.rs:225]: Probably, I don't recall why I did it like this. Will do it at the next opportunity.

**Pull request merged**